src/Spryker/Client/RabbitMq/Model/Consumer/Consumer.php

```diff
@@ -128,6 +133,24 @@ public function collectQueueMessages(AMQPMessage $message)
         $this->collectedMessages[] = $queueReceiveMessageTransfer;
     }
 
+    /**
+     * @param \PhpAmqpLib\Message\AMQPMessage $message $message
+     * @param \Generated\Shared\Transfer\QueueSendMessageTransfer $queueSendMessageTransfer
+     *
+     * @return \Generated\Shared\Transfer\QueueSendMessageTransfer
+     */
+    protected function addApplicationHeaders(AMQPMessage $message, QueueSendMessageTransfer $queueSendMessageTransfer): QueueSendMessageTransfer
+    {
+        $messageProps = $message->get_properties();
```

Comment:
Please do not use abbreviations. Please use `$messageProperties`

Reply:
@stereomon, thank you, updated